awx/ui/client/src/templates/prompt/steps/preview/prompt-preview.partial.html

```diff
@@ -0,0 +1,84 @@
+<div>
+    <div class="Prompt-previewRow--flex">
+        <div class="Prompt-previewRowTitle">{{:: vm.strings.get('prompt.JOB_TYPE') }}</div>
+        <div class="Prompt-previewRowValue">
+            <span ng-if="promptData.prompts.jobType.value.value === 'run'">{{:: vm.strings.get('prompt.PLAYBOOK_RUN') }}</span>
+            <span ng-if="promptData.prompts.jobType.value.value === 'check'">{{:: vm.strings.get('prompt.CHECK') }}</span>
+        </div>
+    </div>
+    <div class="Prompt-previewRow--flex">
+        <div class="Prompt-previewRowTitle">{{:: vm.strings.get('prompt.CREDENTIAL') }}</div>
+        <div class="Prompt-previewRowValue"  style="display: flex">
+            <div class="Prompt-previewTagContainer u-wordwrap" ng-repeat="credential in promptData.prompts.credentials.value">
+                <div class="Prompt-previewTag">
+                    <span ng-switch="promptData.prompts.credentials.credentialTypes[credential.credential_type]">
+                        <span class="fa fa-cloud" ng-switch-when="cloud"></span>
+                        <span class="fa fa-info" ng-switch-when="insights"></span>
+                        <span class="fa fa-sitemap" ng-switch-when="net"></span>
+                        <span class="fa fa-code-fork" ng-switch-when="scm"></span>
+                        <span class="fa fa-key" ng-switch-when="ssh"></span>
+                        <span class="fa fa-archive" ng-switch-when="vault"></span>
+                    </span>
+                    <span ng-bind="credential.name"></span>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="Prompt-previewRow--flex">
+        <div class="Prompt-previewRowTitle">{{:: vm.strings.get('prompt.INVENTORY') }}</div>
+        <div class="Prompt-previewRowValue" ng-bind="promptData.prompts.inventory.value.name"></div>
+    </div>
+    <div class="Prompt-previewRow--flex">
+        <div class="Prompt-previewRowTitle">{{:: vm.strings.get('prompt.LIMIT') }}</div>
+        <div class="Prompt-previewRowValue" ng-bind="promptData.prompts.limit.value"></div>
+    </div>
+    <div class="Prompt-previewRow--flex">
+        <div class="Prompt-previewRowTitle">{{:: vm.strings.get('prompt.VERBOSITY') }}</div>
+        <div class="Prompt-previewRowValue" ng-bind="promptData.prompts.verbosity.value.label"></div>
+    </div>
+    <div class="Prompt-previewRow--noflex">
```

Comment:
For these collapsible panels (and perhaps for all fields on the Prompt Preview), I'm wondering if we should hide the "preview row" if nothing was specified. It looks kind of odd to have any empty fields: 
![image](https://user-images.githubusercontent.com/7010629/35822423-1d200c34-0a61-11e8-8416-f437c941e0e2.png)